### PR TITLE
🐛 fix crash on aws.iam.virtualMfaDevices & logGroup.metricsFilters alarms failure

### PR DESCRIPTION
**Pull request by vjeffrey**
fixes https://github.com/mondoohq/cnquery/issues/3179

fixes https://github.com/mondoohq/cnquery/issues/3178

### CODE REVIEW

File: providers/aws/resources/aws_iam.go [hunk 1]
```diff
@@ -288,7 +288,9 @@ func (a *mqlAwsIam) virtualMfaDevices() ([]interface{}, error) {
 
 	devicesResp, err := svc.ListVirtualMFADevices(ctx, &iam.ListVirtualMFADevicesInput{})
 	if err != nil {
-		return nil, errors.Wrap(err, "could not gather aws iam virtual-mfa-devices")
```
Comment (chris-rock) [inline providers/aws/resources/aws_iam.go:291]: any reason why we not set the error?

Comment (vjeffrey) [reply providers/aws/resources/aws_iam.go:291]: ah ya we should do that, pushing the change now

Comment (chris-rock) [reply providers/aws/resources/aws_iam.go:291]: With https://github.com/mondoohq/cnquery/pull/3191 a plain error would also work. This approach is also fine.

**Review by chris-rock — APPROVED** [review]
Thank you @vjeffrey

**Pull request merged**